README.md

```diff
@@ -95,6 +95,29 @@ fpath+=~/.zfunc
 ```
 
 
+## Docker example
+
+```dockerfile
+FROM python:3.7-alpine
+
+MAINTAINER name <email>
+
+# copy pyproject file
+COPY pyproject.toml .
+
+# install poetry
+RUN pip --no-cache-dir install poetry poetry-setup
+
+# We don't need a virtualenv since is Docker...
+RUN poetry config settings.virtualenvs.create false
```

Comment:
I'd say the artifact that this command creates shoudl be removed after installation. This implies that these three layers that are created by `RUN` directives should be merged. `poetry` itself should also be uninstalled after it did its job, imo. #932 is related.

Reply:
I am sorry, but I don't understand what are you saying... So I uninstall `poetry` and what else?

Reply:
```Dockerfile
RUN pip --no-cache-dir install poetry poetry-setup \
 && poetry config settings.virtualenvs.create false \
 && poetry install \
 && pip uninstall poetry \  # and all of its dependencies ... :-/
 && rm ~/.config/pypoetry
```

that would be it.

Reply:
`poetry-setup` isn't used at all, is it?

Reply:
you are right, the only use case is that you are testing a new dependency so you install with poetry

Reply:
@funkyfuture doesn't this still have the problem of potentially conflicting versions between Poetry's dependencies and app dependencies?

Reply:
indeed. but would these confilcts matter? at the point where poetry is executed it has all its dependencies loaded. if the project installation breaks these, be it so, poetry won't be executed another time. nonetheless i think a standalone script that just does what the `install` command achieves would be the best solution to deploy projects.